docs/Feature-Envy.md

```diff
@@ -2,28 +2,48 @@
 
 ## Introduction
 
-Feature Envy occurs when a code fragment references another object more often than it references itself, or when several clients do the same series of manipulations on a particular type of object.
+`Feature Envy` occurs when a code fragment references another object more often than it references itself, or when several clients do the same series of manipulations on a particular type of object.
```

Comment:
@mvz @chastell let's go with italic for smell names and things like that. objections?